README.md

```diff
@@ -1,2 +1,49 @@
+# Introduce junbae
+
+We introduce Junbae for people using the this vscode extenstion.
+We don't know the gender yet, but i think it will be a pretty cute parrot who will be raised by the person making this program.
+The name is taken from Junbae's name.
+
+# Motivation
+
+We wanna create vscode extension program that does not effective for coding but gives you the satisfication of coding while typing your code.
+It is sure you have satisfication for visual effect but you does not get the help typing code.
+We are referrence to vscode's PowerMode extension program which has been unmaintained since 2017.
+If you wanna get the confidence for typing code, using the Junbae-mode extenstion. We are sure you get the more and more satisfication this effects.
+
 # junbae-mode
+
 Junbae is caique parrot!
+It is activated when you typing keyboard. It will dance to the beat of your typing
+If you wanna get the source code, you visit this github url and you can contribute for the this code if you follow CONTRIBUTING.md file up.
+We hope that many people contribute the visual effect extension program for visual coding and have fun during code. Junbae blesses you!!
```

Comment:
```suggestion
It is activated when you type. It will dance to the beat of your typing
If you wanna get the source code, you visit this GitHub URL and you can contribute to this code if you follow CONTRIBUTING.md file up.
We hope that many people contribute to the visual effect extension program for visual coding and have fun during code. Junbae blesses you!!
```